coding-concepts.js

```diff
@@ -7,49 +7,53 @@
 // There is no need to change any of the code.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//  console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:["tangerine", "magenta", "lilac", "daffodil", "indigo"]
 
-// --------------------1) What will this log?
+// b) Verify and explain: output is 5. its only counting the amount of strings in the array instead of displaying the strings in the array.
 
-const cohort = "LEARN 2023"
-// console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // --------------------1) What will this log?
 
-// --------------------2) What will this log?
+// const cohort = "LEARN 2023"
+//  console.log(cohort.length)
 
-const greeting = "Hello World!"
-// console.log(greeting[4])
+// // a) Your answer: 10
 
-// a) Your answer:
-// b) Verify and explain:
+// // b) Verify and explain: 10. you only see 9 characters but the empty space between "N" and "2" is still counted in the string as an character
 
-// --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+// // --------------------2) What will this log?
 
-// a) Your answer:
-// b) Verify and explain:
+// const greeting = "Hello World!"
+//  console.log(greeting[4])
 
-// --------------------4) What will this log?
+// // a) Your answer: o
+// // b) Verify and explain: index starts at zero so im technically (as a human) am counting 5 not 4 which means whatevers in the 5 index is correct
 
-const weekendDays = ["saturday", "sunday"]
+// // --------------------3) What will this log?
+
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
+//  console.log(languages[index])
+
+// // a) Your answer:"Ruby"
+// // b) Verify and explain: Its counting the strings as individuals in the array 
+
+// // --------------------4) What will this log?
+
+// const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:SATURDAY, SUNDAY
+// // b) Verify and explain:.toUpperCase is a built-in method that uppercases anything its connected to
```

Comment:
So this is not what the output is.  Make sure to run the code to see what is actually returned, and if it's different, explain why (which could take some research).